NitroxPatcher/Patches/Dynamic/FireExtinguisherHolder_TryStoreTank_Patch.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class FireExtinguisherHolder_TryStoreTank_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((FireExtinguisherHolder t) => t.TryStoreTank());
+
+    public static readonly OpCode INJECTION_OPCODE = OpCodes.Callvirt;
+    public static readonly object INJECTION_OPERAND = Reflect.Method((GameObject gameObject) => gameObject.SetActive(default(bool)));
+
+    public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnumerable<CodeInstruction> instructions)
+    {
+        foreach (CodeInstruction instruction in instructions)
+        {
+            yield return instruction;
+            Log.Info(instruction);
+
+            if (instruction.opcode.Equals(INJECTION_OPCODE) && instruction.operand.Equals(INJECTION_OPERAND))
+            {
+                /*
+                 * Injects:  Callback(this, pickupable);
```

Comment:
Same here, but it'd be even better because the exact position is important for the ``Callback``